### ISSUE TITLE
Importer skipping timesteps 

### ISSUE THREAD

**Issue by openSourcerer9000** [HydrologicEngineeringCenter/Vortex#69]
```
[Timestamp('2020-10-16 17:30:00'),
 Timestamp('2020-10-16 18:30:00'),
 Timestamp('2020-10-20 15:30:00'),
 Timestamp('2020-10-20 16:30:00'),
 Timestamp('2020-10-21 23:30:00'),
 Timestamp('2020-10-22 00:30:00'),
 Timestamp('2020-10-23 03:30:00'),
 Timestamp('2020-10-23 04:30:00'),
 Timestamp('2020-10-23 18:30:00'),
 Timestamp('2020-10-23 19:30:00')]
```

![image](https://user-images.githubusercontent.com/61931826/171281835-d83775f0-2b8f-438f-b45a-62fb613542e4.png)
![image](https://user-images.githubusercontent.com/61931826/171281905-3304a59b-0f5c-4f73-9f18-9a8c6cf0a2bb.png)
[InputData.zip](https://github.com/HydrologicEngineeringCenter/Vortex/files/8809397/InputData.zip)

**Comment by danhamill** [comment]
I was also able to reproduce this issue.  There should be 745 records, but vortex only imported 741:

```cmd
08:58:45.499      -----DSS---zclose  Handle 3;  Process: 38776;  File: C:\Temp\vortex\InputData\test.dss
08:58:45.501                         Number records:         741
08:58:45.501                         File size:              183484  64-bit words
08:58:45.502                         File size:              1433 Kb;  1 Mb
08:58:45.502                         Dead space:             0
08:58:45.503                         Hash range:             8192
08:58:45.504                         Number hash used:       533
08:58:45.505                         Max paths for hash:     3
08:58:45.505                         Corresponding hash:     40
08:58:45.506                         Number non unique hash: 0
08:58:45.507                         Number bins used:       533
08:58:45.507                         Number overflow bins:   0
08:58:45.507                         Number physical reads:  962
08:58:45.508                         Number physical writes: 5060
08:58:45.508                         Number denied locks:    0
```

**Comment by tombrauer** [comment]
I'm looking at Delta2020.nc in Panoply Viewer and the time axis seems to have some discontinuities:

**Comment by openSourcerer9000** [comment]
Mmmm so `valid_time` is important? I was doing everything by coord var `time`, including interpolating some gaps, which seemed to have dropped the values of the auxillary coords for those timesteps. I'll fill out `valid_time` and hopefully that clears up the consistent timestep drops.

I think I'll add `legit_time_yo` and `@therealtimeauxillarycoordvar` as well just to cover all the bases.

**Comment by openSourcerer9000** [comment]
Yes, duplicated the `time` coordinate to `valid_time` cleared that up. It still randomly skips timesteps, so I will go ahead and leave this open, but I am at least able to work around that by rerunning importer until it fills out the DSS.

**Comment by danhamill** [comment]
@openSourcerer9000 are you generating these netcdf files your self?

**Comment by danhamill** [comment]
Once idea I had to help define the NETCDF structure required to vortex to properly import to DSS is to develop an additional utility that will convert a dss file to NETCDF format.  This would provide a straight forward approach to users developing their own NETCDF files.  Might be outside the scope of Vortex, but I personally find the CF conventions hard to follow since many datasets only partially conform to the CF convention.

**Comment by tombrauer** [comment]
I designed Vortex with this in mind. You'll notice that the option to specify pathname parts only become available after a file is selected. So, if you were to select something like a *.nc file for output, you may see options related to that file type. To add NetCDF write capability to Vortex a `NetcdfDataWriter` implementation would need to be added that extends `DataWriter`.

**Comment by tombrauer** [comment]
@openSourcerer9000 I can't reproduce the timestep skipping issue. Do you have a dataset that reliably reproduces?
